neon_core/skills/__init__.py

```diff
@@ -23,10 +23,18 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from neon_utils.skills.mycroft_skill import PatchedMycroftSkill
 from neon_core.skills.neon_skill import NeonSkill
 from neon_core.skills.fallback_skill import NeonFallbackSkill
 from neon_core.skills.decorators import intent_handler, intent_file_handler, \
-    resting_screen_handler, conversational_intent, skill_api_method
+    resting_screen_handler, conversational_intent
+
+import mycroft.skills.core
```

Comment:
this is not enough, mycroft provides the skill import in a few places, so depending on where the skill class is imported from it might be patched or not

not sure if i missed some place, but in monkey patcher skill i have
```
import mycroft

mycroft.skills.mycroft_skill.MycroftSkill = MycroftSkill
mycroft.skills.MycroftSkill = MycroftSkill
mycroft.skills.core.MycroftSkill = MycroftSkill
```

Reply:
in a future PR i think we will want to avoid this and go the WrapperClass route like in https://github.com/NeonJarbas/neon_audio/commit/f3e7363bcea684159c9bfaae1af2ffeb809a497b

will open PR across all the relevant repos with this wrapper approach, other than the missing patch above i think this is good

Reply:
Good catch; I'll add that and merge.

Regarding the wrapper, how would a skill know to import `PatchedMycroftSkill` without patching the imports like this?